### PR TITLE
Added Makefile stub.

### PR DESCRIPTION
**Pull request by thom-at-redhat**
"make init" works
"make docs" works (depends on "make init", then runs "make html" in docs
subdirectory)

Added JBDS (Eclipse) ignores also

### CODE REVIEW

**Comment by blrm** [comment]
:+1:

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#
+#
+# $Id: 4c1a8a61a4654b13ab641bde241c19ffc531101d $
+#
+.PHONY: 2to3 coverage docs init install lint test
+
+2to3:
+	echo futurize
```
Comment (jamesooden) [inline Makefile]: Why are we just echoing them to the screen?

File: pages/regions/cancel_deployment_modal.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#
+# $Id$
+#
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class CancelDeploymentModalPage(Base):
+    """Cancel Deployment Modal Window:
```
Comment (jamesooden) [inline pages/regions/cancel_deployment_modal.py]: I would move the text to the next line for readability reasons.

File: pages/regions/cancel_deployment_modal.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    1) Modal Header
```
Comment (jamesooden) [inline pages/regions/cancel_deployment_modal.py]: Some of these steps don't make sense to me...they don't seem like steps the way they are worded.   I'm confused which is not to hard to do.

File: pages/regions/cancel_deployment_modal.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    _page_title = "QuickStart Cloud Installer"
+
+    _exit_and_delete_button_locator = (
+        By.XPATH, "//button[contains( text(), 'Exit and Delete' )")
+
+    @property
+    def exit_and_delete(self):
+        return self.selenium.find_element(*self._exit_and_delete_button_locator)
+
+    _exit_and_save_button_locator = (
```
Comment (jamesooden) [inline pages/regions/cancel_deployment_modal.py]: Most of the other selenium classes puts all the locators up at the top of the class.   I can see arguments for either approach...I just think we should be consistent.

File: pages/regions/modal_dialog.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+# $Id$
+#
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class ModalDialogPage(Page):
+    """Unknown:
+    1) Modal header
```
Comment (jamesooden) [inline pages/regions/modal_dialog.py]: I don't know what you are listing here.

File: pages/regions/modal_dialog.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
+#
+# $Id$
+#
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class ModalDialogPage(Page):
```
Comment (jamesooden) [inline pages/regions/modal_dialog.py]: This class doesn't seem to do anything.

File: pages/regions/nav_bar_outer.py [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+    2) Red Hat Access dropdown menu
+
+    3) User dropdown menu
+
+    Black background, white lettering"""
+
+    _page_title = "QuickStart Cloud Installer"
+
+    # Navbar-header
```
Comment (jamesooden) [inline pages/regions/nav_bar_outer.py]: Are these all comments you are going to fill out later?

File: pages/regions/page_header.py [hunk 8]
```diff
@@ -0,0 +1,30 @@
+#
+# $Id$
+#
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class PageHeader(Page):
```
Comment (jamesooden) [inline pages/regions/page_header.py]: What does this do?

**Pull request closed without merging**